Library/Formula/qjackctl.rb

```diff
@@ -0,0 +1,118 @@
+class Qjackctl < Formula
+  desc "simple Qt application to control the JACK sound server daemon"
+  homepage "http://qjackctl.sourceforge.net"
+  url "https://downloads.sourceforge.net/qjackctl/qjackctl-0.4.1.tar.gz"
+  sha256 "98024e1ea9f55ac16c290feda051d6494b2261e83f6a918f0fa583b509e2bfd1"
+
+  depends_on "autoconf" => :build
+  depends_on "qt"
+  depends_on "jack"
+
+  patch :DATA
```

Comment:
Well this patch is a compilation of different commits that are already contained in upstream. The respective commit hashes are already contained in the patch. I will add additional comments why a backport is necessary from my point of view.